tests/test_parsers.py

```diff
@@ -30,10 +31,12 @@ def test_gelfparser_parse_raw_file(gelf_logger):
     gelf_logger.info('{"username": "foo"}')
     gelf_logger.info('{"username": ""}')
 
-    events = list(parser.parse(open(gelf_logger.handlers[0].stream.name)))
-    assert len(events) == 2
-    assert events[0] == '{"username": "foo"}'
-    assert events[1] == '{"username": ""}'
+    with open(gelf_logger.handlers[0].stream.name) as stream:
+        events = list(parser.parse(stream))
+
+        assert len(events) == 2
+        assert events[0] == '{"username": "foo"}'
+        assert events[1] == '{"username": ""}'
```

Comment:
This block doesn't require indentation:

```suggestion
    assert len(events) == 2
    assert events[0] == '{"username": "foo"}'
    assert events[1] == '{"username": ""}'
```